TooltipDecorator/useTooltip.js

```diff
@@ -0,0 +1,266 @@
+/* global MutationObserver ResizeObserver */
+
+import {forProp, forward, handle, not} from '@enact/core/handle';
+import {Job} from '@enact/core/util';
+import {useI18nContext} from '@enact/i18n/I18nDecorator';
+import {FloatingLayerBase} from '@enact/ui/FloatingLayer';
+import ri from '@enact/ui/resolution';
+
+import {Tooltip, defaultArrowAnchor, defaultDirection} from './Tooltip';
+import {adjustDirection, adjustAnchor, calcOverflow, getLabelOffset, getPosition} from './util';
+
+let currentTooltip; // needed to know whether or not we should stop a showing job when unmounting
+
+import useHandlers from '@enact/core/useHandlers';
+import {useCallback, useEffect, useLayoutEffect, useRef, useState} from 'react';
+
+function getDirectionAnchor (tooltipPosition, tooltipType) {
```

Comment:
```suggestion
function getTooltipDirection (tooltipPosition, tooltipType) {
```

Reply:
Let's convert this to an arrow function.

Reply:
👍